evap/results/tests/test_views.py

```diff
@@ -26,7 +26,7 @@ def setUpTestData(cls):
 
 
 class TestResultsSemesterCourseDetailView(ViewTest):
-    url = '/results/semester/2/course/21'
+    url = '/results/semester/1/course/1'
```

Comment:
the course still is created - why doesn't it work now?
i would be happy creating the data with mommy, but i couldn't get the test running. please feel free to change everything.